### ISSUE TITLE
Tasks processed counting up even when a task fails instantly

### ISSUE THREAD

**Issue by cryptobench** [golemfactory/yagna-triage#136]
I was wondering if the output from `golemsp status` in `total processed` counted up in value even on tasks that failed to compute instantly or during the task, so I decided to test it out with Mattias.

What we found is that even we have the task fail instantly on Mattias's provider, then it counts it as a task processed.
He started with 57 total processed and after my testing he has over 114 total processed tasks, and none of them completed sucessfully. Here's how it looks on my end.
![image](https://user-images.githubusercontent.com/33448819/124133715-65980f80-da82-11eb-9dba-f8f23751064f.png)

**This especially confuses our providers on the network because they see that they completed tasks overnight by watching the `total processed` amount, and then their wallet doesn't match because it hasn't gone up, because of course the requestor doesn't want to pay for failed tasks.** 

**We are wasting a lot of resources investigating "payment bugs" because we judge the `total processed` value as an indicator of a succesfull task and with a task sucessfully completed should a payment also come in.**

What I propose that we do is have the same CLI output as Clay/Brass had with the following values:

```
Tasks completed: n
Tasks timed out: n
Tasks failed: n
```
This would give a clear indicator on what happened on the provider side with the task that has flowed in.

**Comment by shadeofblue** [comment]
well, I'm unsure if the provider has an idea whether the task failed or not ... plus, it's up to the requestor to decide whether they want to pay for a failed task or not ... currently, yapapi and yajsapi don't make a distinction and pay for all received invoices related to negotiated agreements for which the activities have been started without discriminating for failed ones ... that's what @mbenke said today -> his node is paying for everything even if the providers fail instantly ...

**Comment by cryptobench** [comment]
I just think this is a major UX flaw that it counts as a task processed even though it fails instantly on the provider side. Our users thinks the network isn't working properly and they're not getting paid for the work they've done. It's gonna give the network a bad reputation because it seems like the "payment bug" has been around for almost half a year and continues to be.

**Comment by nieznanysprawiciel** [comment]
Like @shadeofblue said, it is not so obvious how can we solve this problem.
One of potential solutions will be discussed as part of this issue: https://github.com/golemfactory/yagna/issues/1487

**Comment by tworec** [comment]
Neat finding by both of you @cryptobench and @mat7ias 

The semantics of `processed` seems to be `processed successfully` intuitively.
As the proper solution is not so obvious (I've not investigated it myself, just basing on what @shadeofblue  and @nieznanysprawiciel both are saying) maybe the workaround with renaming this counter will do initially?
We can rename from `total processed` to `total processed (inc. failures)`

**Comment by cryptobench** [comment]
Yeah I think that could be a fine temporary solution to the problem, because providers thinks there's a payment bug running around on the network.

**Comment by tworec** [comment]
@jamuszyn can you pls give us advice on how to rework and improve on UX here?
I've also asked Piotr Słowik to put his two cents.

**Comment by nieznanysprawiciel** [comment]
Blocked by: https://github.com/golemfactory/yagna/issues/1545
